src/Lazy.elm

```diff
@@ -17,7 +17,7 @@ module Lazy
 @docs apply, andThen
 -}
 
-import Native.Lazy
+import Native.Lazy exposing (..)
```

Comment:
Ah, I didn't know that the namespace for native modules should have special considerations apart from fully Elm ones. It has been reverted!